clang/lib/3C/ConstraintVariables.cpp

```diff
@@ -982,26 +982,25 @@ FunctionVariableConstraint::allocateParamPair(const clang::QualType &QT,
                                               std::string N, ProgramInfo &I,
                                               const clang::ASTContext &C,
                                               std::string *InFunc,
-                                              bool VarAtomForChecked) {
+                                              bool HasItype) {
```

Comment:
I am wondering if `HasItype` is information that could be derived from the other parameters you already have, e.g., the `DeclaratorDecl` `D` ?